tests/test_logger.py

```
@@ -2,272 +2,287 @@
-    
-    @patch.dict('os.environ', {'LOG_FILE_LEVEL': 'INVALID'})
+
+    @patch.dict("os.environ", {"LOG_FILE_LEVEL": "INVALID"})
     def test_logger_invalid_env_level_uses_default(self):
         """Test that invalid environment log level uses default"""
         # Reset singleton to pick up new environment
         Logger._instance = None
-        
+
         # This should not raise an exception and should use defaults
         logger = Logger()
```

Ruff (0.13.1)

Local variable `logger` is assigned to but never used

Remove assignment to unused variable `logger`

(F841)